Sources/RxMoya/RxMoyaProvider.swift

```diff
@@ -21,7 +21,10 @@ open class RxMoyaProvider<Target>: MoyaProvider<Target> where Target: TargetType
 
         // Creates an observable that starts a request each time it's subscribed to.
         return Observable.create { [weak self] observer in
```

Comment:
I think that would be the expected behavior, i produced the provider inside a method since i only needed it once. I can see someone else also doing the same thing.